Foundation/NSPropertyList.swift

```diff
@@ -113,6 +113,8 @@ internal func _expensivePropertyListConversion(input : AnyObject) -> Any {
         return data
     } else if let number = input as? NSNumber {
         return number
+    } else if let url = input as? NSURL {
```

Comment:
I guess I was thinking of using this as a general purpose conversion function until we get our bridging story straightened out.